tests/actions.go

```diff
@@ -504,6 +508,9 @@ func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 		return fmt.Errorf("failed to create statefulset webhook and configuration : %v, %s", err, string(res))
 	}
 
+	// wait for all apiservices are avaiable
+	// '-l a!=b' is a workaround solution for '--all' flag which is introduced only in kubectl 1.14+
+	oa.runKubectlOrDie("wait", "--for=condition=Available", "apiservices", "-l", "a!=b")
```

Comment:
wait for all API services are available after webhook API service is registered, otherwise, discovery API request will fail